staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go

```diff
@@ -26,9 +26,9 @@ import (
 // SetCRDCondition sets the status condition.  It either overwrites the existing one or
 // creates a new one
 func SetCRDCondition(crd *CustomResourceDefinition, newCondition CustomResourceDefinitionCondition) {
+	newCondition.LastTransitionTime = metav1.NewTime(time.Now())
```

Comment:
shouldn't we have some deepequal comparison here?

Reply:
I don't follow...why do we need to compare here?

Reply:
the timestamp should not change if the condition being set already exists and has the same status